### PR TITLE
Initial implementation of drone-chef-supermarket

### PR DESCRIPTION
**Pull request by jmccann**
This is my initial implementation of a supermarket plugin based on code from https://github.com/drone-plugins/drone-chef

### CODE REVIEW

**Comment by tboerger** [comment]
So you want to rename it to drone-supermarket now?

**Comment by jmccann** [comment]
Not necessarily.  I started writing the code as `drone-supermarket` and then remembered you had this repo and then forgot to update my code.  😨 

I'll work on getting it back to `drone-chef-supermarket`.

**Comment by tboerger** [comment]
Still named drone-supermarket on multiple places :)

**Comment by jmccann** [comment]
@tboerger I think I got them all now? 😨

**Comment by jmccann** [comment]
FYI, I've started using it @ https://hub.docker.com/r/jmccann/drone-chef-supermarket/ and seems OK so far

File: DOCS.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
-The following is a sample configuration in your .drone.yml file:
+Example
+=======
 
+### Minimal Definition
+This will upload the cookbook to a supermarket server
 ```yaml
 deploy:
-  chef_supermarket:
+  chef:
```
Comment (tboerger) [inline DOCS.md]: just use `chef_supermarket` and it will be replaced with `plugins/drone-chef-supermarket`

**Comment by tboerger** [comment]
Just fix the example, than it LGTM

**Pull request merged**